cpu/kinetis_common/isr_kinetis.c

```diff
@@ -105,6 +105,7 @@ WEAK_DEFAULT void isr_i2s0_rx(void);
 WEAK_DEFAULT void isr_i2s0_tx(void);
 WEAK_DEFAULT void isr_llwu(void);
 WEAK_DEFAULT void isr_lptmr0(void);
+WEAK_DEFAULT void isr_lpuart0(void);
```

Comment:
This is the weak default for the function used in vectors.c (https://github.com/gebart/RIOT/blob/pr/frdm-k22f/cpu/k22f/vectors.c#L76) for when the LPUART module is not used by the UART driver.